dir.props

```diff
@@ -1,5 +1,5 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" InitialTargets="CheckRoslyn" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+  <Import Condition="Exists('..\dir.props') And '$(RepoLocalBuild)'==''" Project="..\dir.props" />
```

Comment:
I don't want to suggest there is something closed -- `OpenBuild`.

`RepoLocal` is applicable anywhere the repo is cloned. Keeping the name!